src/modules/tracker.js

```diff
@@ -79,6 +85,7 @@ function applyParamsAsString(parameters, options) {
 class Tracker {
   constructor(options) {
     this.options = options;
+    this.options.beaconMode = (options && options.beaconMode === false) ? false : true; // Defaults to 'true'
```

Comment:
I would be careful here modifying passed in parameters ... should probably set `this.options = {...options}`